### ISSUE TITLE
Clicking on the Item in Sidebar should open top by default

### ISSUE THREAD

**Issue by aswinmohanme** [elixir-lang/ex_doc#895]
The current behavior is not a good user experience. Having to open the sidebar menu and then select top is really redundant, and after some time annoying.

I have read the previous issues and found that the community has mixed feelings about this, but it would be great if we could work out a solution.

**Comment by aswinmohanme** [comment]
<img width="361" alt="frame 5" src="https://user-images.githubusercontent.com/20600674/45396403-0eb6b680-b659-11e8-8342-ef83d515aaa6.png">

Clicking on the title should go to the top by default and open up the sidebar, and when the + is clicked it should stay on the current page and open up the sidebar

**Comment by josevalim** [comment]
I actually like your proposal a lot. We always thought about adding something to the beginning but at the end provides a better user experience. The issue though is that it conflicts with #834, so we need a solution that works nicely in both cases.

**Comment by michallepicki** [comment]
I very like @aswinmohanme 's idea.

Another issue I have with our current menu (similar to #953 ) is that when a module (or another page) has content, the default way to navigate to this page is to click "Top", which uses an anchor link `#content`. After refreshing a page that was opened this way (for example, after closing the browser and restoring pages from previous session) the browser scroll position wouldn't be preserved.

I would really like if the default way of opening a page wouldn't use anchor links.

**Comment by josevalim** [comment]
I pushed a feature were we now have a "goto" arrow close to the sidebar root entries. Clicking on the arrow sends you directly to that page. This arrow appears only on mouse over to not pollute the UI:

![image](https://user-images.githubusercontent.com/9582/51472910-2de9f100-1d7b-11e9-8749-315748d7bf31.png)

Clicking on the menu still expands up the menu. We also fixed #953 to consistently expand the menu.
